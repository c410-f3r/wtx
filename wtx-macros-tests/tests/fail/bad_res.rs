#[wtx_macros::pkg(data_format(json), id(FooId))]
mod pkg {
  #[derive(Debug)]
  #[pkg::req_data]
  pub struct Req;

  #[pkg::res_data]
  struct ResDSA;
}

fn main() {
}
