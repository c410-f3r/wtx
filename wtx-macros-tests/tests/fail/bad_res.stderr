error: Response data must end with the `Res` suffix.
 --> tests/fail/bad_res.rs:8:10
  |
8 |   struct ResDSA;
  |          ^^^^^^

error: aborting due to 1 previous error

